tests/Aspire.Workload.Tests/AppHostTemplateTests.cs

```diff
@@ -0,0 +1,42 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+using Xunit.Abstractions;
+using System.Text.RegularExpressions;
+
+namespace Aspire.Workload.Tests;
+
+public partial class AppHostTemplateTests : WorkloadTestsBase
+{
+    public AppHostTemplateTests(ITestOutputHelper testOutput)
+        : base(testOutput)
+    {
+    }
+
+    [Fact]
+    public async Task EnsureProjectsReferencing8_1_0WorkloadCanBuild()
```

Comment:
Should this be named `EnsureProjectsReferencing8_1_0AppHostWithNewerWorkloadCanBuild`?